flatpak/io.github.lact-linux.yaml

```
@@ -0,0 +1,39 @@
+app-id: io.github.lact-linux
```

Appid is wrong, you need to specify the entire repo like:
```suggestion
app-id: io.github.ilya_zlobintsev.LACT
```